apps/frontend/src/pages/access-management/styles.module.css

```
@@ -0,0 +1,22 @@
+.content {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	padding-left: 10%;
```

According to Figma design
```suggestion
	padding: 32px 80px;
```

---

Or maybe even it would be better to move this padding to the `CommonLayout` component, as it is the same for all pages